tests/e2e/tests/e2e_happy_path/DevWorkspaceHappyPath.spec.ts

```diff
@@ -55,7 +54,7 @@ const SpringAppLocators = {
 
 // this test checks only workspace created from "web-nodejs-sample" https://github.com/devfile/devworkspace-operator/blob/main/samples/flattened_theia-next.yaml.
 suite('Workspace creation via factory url', async () => {
-    // let factoryUrl : string = `${TestConstants.TS_SELENIUM_DEVWORKSPACE_URL}`;
+    let factoryUrl : string = `${TestConstants.TS_SELENIUM_BASE_URL}/f?url=${TestConstants.TS_SELENIUM_DEVWORKSPACE_URL}`;
```

Comment:
It will not work on PR checker because TS_SELENIUM_DEVWORKSPACE_URL includes TS_SELENIUM_BASE_URL like here:
        - name: TS_SELENIUM_DEVWORKSPACE_URL
          value: http://che-eclipse-che.apps.ci-op-dhvtilit-c682c.origin-ci-int-aws.dev.rhcloud.com/#https://github.com/che-samples/java-spring-petclinic/tree/devfilev2

Reply:
Just remove comment from factoryUrl

Reply:
Oh ok